### DISCUSSION TITLE
coerce_int96_timestamp_unit warning

### DISCUSSION THREAD

**Discussion by mycaule** [aws/aws-sdk-pandas#1763] [Q&A]
When reading Parquet with `wr.s3.read_parquet` I have multiple warnings of this form, 

How do I resolve this issue thank you ?

```
_read_parquet.py:47
coerce_int96_timestamp_unit is not supported in pyarrow 2 and below
```

**Comment by malachi-constant** [comment]
Hi @mycaule are you able to upgrade `pyarrow` to` >=3.x.x`?

**Reply by mycaule** [reply]
I am having this error on `airflow==2.2.2` which has `pyarrow==4.0.1`.

https://raw.githubusercontent.com/apache/airflow/constraints-2.2.2/constraints-3.7.txt

**Reply by malachi-constant** [reply]
It's possible our warning is inaccurate then, I will try to replicate with `4.0.1` and go from there.

**Reply by malachi-constant** [reply]
@mycaule Can you tell me which version of the pandas sdk (`awswrangler`) you are using?

**Reply by mycaule** [reply]
`awswrangler==2.12.1`, I can also say that I am using MWAA on AWS.

**Reply by malachi-constant** [reply]
I can confirm that this is replicated with `pyarrow 4` as a dependency. So the warning is incorrect, however this is [still raising a `TypeError`](https://github.com/aws/aws-sdk-pandas/blob/829c306456cb3caea7c677d9a7935d514b7af7f2/awswrangler/s3/_read_parquet.py#L45) and not sure there is anything else you can do outside of upgrading to a version of airflow that supports pyarrow 6 or greater.

**Reply by mycaule** [reply]
MWAA currently runs airflow=2.2.2 maximum

**Reply by malachi-constant** [reply]
Understood, unfortunately there isn't anything we can do here until support is added for libraries that work with pyarrow 6 and above.

**Reply by mycaule** [reply]
Do you think I can change the `coerce_int96_timestamp_unit` parameter to something else?

**Reply by malachi-constant** [reply]
Is the question if you can change its value or if there's a different parameter that can be used to set how timestamps are inferred when creating a the parquet file? For the latter i'd have to do some investigation into what pyarrow supported at `4.0.1`
